integration/admin-client/src/main/java/org/keycloak/admin/client/resource/OrganizationsResource.java

```diff
@@ -108,8 +108,9 @@ List<OrganizationRepresentation> search(
      *              the method returns all organizations whose name or (domains) partially match the {@code search} param.
      * @param first the position of the first result to be processed (pagination offset). Ignored if negative or {@code null}.
      * @param max the maximum number of results to be returned. Ignored if negative or {@code null}.
-     * @param briefRepresentation if {@code true} the full representation is to be returned. Otherwise, only the basic fields are returned.
+     * @param briefRepresentation if {@code true} the full representation is to be returned. Otherwise, only the basic fields are returned. The parameter is supported since Keycloak 26.1
      * @return a list containing the matched organizations.
+     * @since Keycloak 26.1. Use method {@link #search(String, Boolean, Integer, Integer)} for the older versions of the Keycloak server
```

Comment:
Shouldn't `@since` tag contain just version number?

Reply:
That's good point. We had same conversation about this with
@jonkoops 
few months ago . Found it on GH here: https://github.com/keycloak/keycloak/pull/32980#discussion_r1762608227 .

We decided to go with `@since` also for indicating the server version as it seems the definition of the `@since` tag is not so strict to enforce the specific format.

I've originaly added some simple text like `since` , but Jon proposed to used `@since` . Basically, we need to specify from which server version the particular endpoint (or parameter) is supported rather than version of the client itself where this endpoint was introduced.